src/screens/UserProfile.tsx

```
@@ -50,6 +51,15 @@ const UserProfile: React.FC = () => {
 		}
 	}, [userData, documents]);
 
+	// Extract nested ternary for phone number display
+	const shouldMaskPhoneNumber = shouldMaskField('phoneNumber') || shouldMaskField('mobile');
```

ESLint

[error] Replace `·` with `⏎↹↹`

(prettier/prettier)